packages/collector/test/tracing/cloud/aws/sqs/test.js

```diff
@@ -144,13 +168,13 @@ mochaSuiteFn('tracing/cloud/aws/sqs', function() {
 
     function verify(receiverControls, response, apiPath, withError, isBatch) {
       if (withError === 'sender') {
-        expect(response.data).to.equal("MissingRequiredParameter: Missing required key 'MessageBody' in params");
+        expect(response.data).to.match(/MissingRequiredParameter: Missing required key 'MessageBody' in params/);
       } else {
         return retry(() => {
           if (isBatch) {
             verifyResponseAndBatchMessage(response, receiverControls);
           } else {
-            verifyResponseAndMessage(response, receiverControls);
+            return verifyResponseAndMessage(response, receiverControls);
```

Comment:
you are right damn. my intention was to add `await`, not return... and even with `await` I am not entirely sure if it will solve the issue.
I will review it, thanks!

Reply:
`verifyResponseAndMessage` is synchronous. No await needed. (We are already inside the `retry` at this point.)